Tests/tests_end_to_end/content/xsoar_saas/test_e2e_xsoar_saas.py

```diff
@@ -0,0 +1,226 @@
+from datetime import timedelta
+from datetime import datetime
+
+import pytest
+from _pytest.fixtures import SubRequest
+import requests
+from demisto_client.demisto_api.models.feed_indicator import FeedIndicator
+from demisto_sdk.commands.common.clients import XsoarSaasClient
+from Tests.tools import get_integration_params
+from Tests.scripts.utils.log_util import install_logging
+from Tests.scripts.utils import logging_wrapper as logging
+from Tests.tests_end_to_end.client_utils import (
+    get_integration_instance_name,
+    get_fetched_incident,
+    save_integration_instance,
+    is_incident_state_as_expected,
+    is_playbook_state_as_expected,
+    save_incident,
+    save_playbook,
+    save_indicators
+)
+
+install_logging('e2e-xsoar-saas.log', logger=logging)
+
+
+@pytest.fixture()
+def available_indicators(xsoar_saas_client: XsoarSaasClient) -> list[str]:
+    """
+    Gets the available indicators in XSOAR-NG.
+    """
+    return [indicator.get("value") for indicator in xsoar_saas_client.list_indicators().get("iocObjects")]
+
+
+def test_edl(request: SubRequest, xsoar_saas_client: XsoarSaasClient, available_indicators: list[str]):
+    """
+    Given:
+        - indicators in xsoar-ng
+        - long-running EDL instance
+    When:
+        - Trying to query the URL of edl
+    Then:
+        - make sure that indicators are returned in the response of edl instance
+    """
+    feed_indicators = [
+        FeedIndicator(value=value, type=_type, score=score) for value, _type, score in
+        [("1.1.1.1", "IP", 0), ("2.2.2.2", "IP", 0), ("3.3.3.3", "IP", 0)]
+    ]
+
+    with save_indicators(xsoar_saas_client, indicators=feed_indicators):
+        integration_params = get_integration_params(
+            request.config.option.integration_secrets_path, instance_name="edl_e2e_instance"
+        )
+        username = integration_params["credentials"]["identifier"]
+        password = integration_params["credentials"]["password"]
+
+        with save_integration_instance(
+            xsoar_saas_client,
+            integration_params=integration_params,
+            integration_id="EDL",
+            is_long_running=True,
+            instance_name=get_integration_instance_name(integration_params, default="EDL")
+        ) as edl_instance_response:
+            instance_name = edl_instance_response.get("name")
+
+            edl_response = xsoar_saas_client.do_long_running_instance_request(
+                instance_name, username=username, password=password
+            )
+            assert edl_response.text, f'could not get indicators from url={edl_response.request.url} from ' \
+                f'instance {instance_name} with available indicators={available_indicators},' \
+                f' status code={edl_response.status_code}, response={edl_response.text}'
+
+
+def test_taxii2_server(
```

Comment:
same for this name. it should be clear what we are checking from the name of the test